src/brogue/Grid.c

```diff
@@ -63,7 +64,7 @@ void fillGrid(short **grid, short fillValue) {
 }
 
 // Highlight the portion indicated by hiliteCharGrid with the hiliteColor at the hiliteStrength -- both latter arguments are optional.
-void hiliteGrid(short **grid, color *hiliteColor, short hiliteStrength) {
+void hiliteGrid(short **grid, const color *hiliteColor, short hiliteStrength) {
```

Comment:
Since I have ensured that all colors that can be defined as const are defined as const I had to update many functions to declare that they take const colors.

Reply:
was this required for something in the framework? just be careful in making a 3k/2k diffs have more concerns than they need to

Reply:
It's not per-se required but I tidied up Globals.h/.c as part of this work by setting `const` on all variables that were conceptually `const`. That tidy-up required adding a few `const`s that should have been there anyway into color functions. I understand the fact that this could be a separate commit (which I'll do if you like) but I think it's pretty low risk.